docs/CustomResources.md

```diff
@@ -244,7 +244,8 @@ metadata:
   name: example
 spec:
   replicas: 3
-  clusterMasterRef: example-cm
+  clusterMasterRef: 
```

Comment:
dont think so this is correct. the previous example is correct and we have been using it for a while now. Can you share your configurations or which version of splunk-operator you are using?

Reply:
I've updated the details in description, could you take a look?

Reply:
Oh yes, you are right. Sorry for the trivial mistake and thanks for catching it!